tsdb/challenges/default.json

```
@@ -53,7 +53,10 @@
         },
         {%- else %}
         {
-          "operation": "delete-index"
+          "operation": {
+            "operation-type": "delete-index",
+            "only-if-exists": false
```

This shouldn't be needed anymore as of elastic/rally#1580

```suggestion
            "operation-type": "delete-index"
```